packages/backend/package.json

```diff
@@ -1,6 +1,43 @@
 {
-  "name": "frontend",
-  "version": "0.0.0",
-  "scripts": {},
-  "devDependencies": {}
+  "name": "backend",
+  "version": "1.0.0",
+  "scripts": {
+    "start": "nodemon dist/index.js",
+    "dev": "concurrently \"npx tsc --watch\" \"nodemon -q dist/index.js\"",
```

Comment:
il nuovo ts lo fa da solo se non sbaglio watch and serve

Reply:
Non ho trovato nessuna referenza a riguardo, senza il "--watch" esce il processo

<img width="585" alt="Screenshot 2022-11-15 at 09 59 42" src="https://user-images.githubusercontent.com/39830028/201875728-9cb5848c-8a12-4e09-8a26-4485a90977bb.png">